### DISCUSSION TITLE
Operations - How should we do things?

### DISCUSSION THREAD

**Discussion by Icedcool** [BanklessDAO/bankless-dao-website#47] [Q&A]
Starting the discussion:
How do we do things? How are repos & access distributed? How are tasks “assigned”? How do we properly evaluate scope of large projects? What expectations do you have for the Guild as a coordination layer? Does the guild have inherent roles? Do we want to use Github Projects or Notion? Etc.

More opinions make this better! 
What does everyone think?

**Comment by pnpancholi** [comment]
In my opinion, dividing it project-wise would be great. Once code-base(s) start to grow, there will always be some ppl working as "core" devs who continuously work on it, or have been on it for a while. I think they can fulfill the role of "staff engineers" eventually, as they can contribute by helping others and managing even without spending most time writing code. GitHub projects are very appealing to me here, Notion is great but it can be easy to clutter it and make it work against us. If it's only relevant to that particular project/repo. then having discussions, tasks, assignments, etc attached to it can very helpful to maintain it. These were my 2 cents on it. Let's hear it from other ppl
